2.7/s2i/bin/run

```diff
@@ -35,6 +35,11 @@ function get_default_web_concurrency() {
   echo $default
 }
 
+APP_HOME=$(readlink -f "${APP_HOME:-.}")
+# Change the working directory to APP_HOME
+PYTHONPATH=$(pwd):$PYTHONPATH
```

Comment:
Unless we're sure that `PYTHONPATH` is set and non-empty at this point, this might introduce a trailing colon, which is not safe with PYTHONPATH as far as I know.